### ISSUE TITLE
Quarkus native checks failure

### ISSUE THREAD

**Issue by squakez** [apache/camel-k#4723]
Most of the time the nightly or PR checks fail with:
```
🍺  /usr/local/Cellar/docker/24.0.5: 13 files, 32.2MB
time="2023-09-01T03:49:37Z" level=info msg="starting colima"
time="2023-09-01T03:49:37Z" level=info msg="runtime: docker"
time="2023-09-01T03:49:37Z" level=info msg="preparing network ..." context=vm
time="2023-09-01T03:49:37Z" level=info msg="creating and starting ..." context=vm
time="2023-09-01T03:49:37Z" level=info msg="Terminal is not available, proceeding without opening an editor"
time="2023-09-01T03:49:37Z" level=warning msg="QEMU binary \"/Users/runner/.colima/_wrapper/4e1b408f843d1c63afbbdcf80c40e4c88d33509f/bin/qemu-system-x86_64\" is not properly signed with the \"com.apple.security.hypervisor\" entitlement" error="failed to run [codesign --verify /Users/runner/.colima/_wrapper/4e1b408f843d1c63afbbdcf80c40e4c88d33509f/bin/qemu-system-x86_64]: exit status 1 (out=\"/Users/runner/.colima/_wrapper/4e1b408f843d1c63afbbdcf80c40e4c88d33509f/bin/qemu-system-x86_64: code object is not signed at all\\nIn architecture: x86_64\\n\")"
time="2023-09-01T03:49:37Z" level=warning msg="You have to sign the QEMU binary with the \"com.apple.security.hypervisor\" entitlement manually. See https://github.com/lima-vm/lima/issues/1742 ."
time="2023-09-01T03:49:37Z" level=info msg="Attempting to download the image" arch=x86_64 digest="sha512:f761b807fe9ba345968df72c07f8c5abcae0c4a44976fe5595c0ff748ef693841221a70e663986c700b027cea32b7cac24d5490d4c721593c39f2b8840c362a2" location="https://github.com/abiosoft/alpine-lima/releases/download/colima-v0.5.5/alpine-lima-clm-3.18.0-x86_64.iso"
Downloading the image (alpine-lima-clm-3.18.0-x86_64.iso)

315.00 MiB / 315.00 MiB (100.00%) ? p/stime="2023-09-01T03:49:42Z" level=info msg="Downloaded the image from \"[https://github.com/abiosoft/alpine-lima/releases/download/colima-v0.5.5/alpine-lima-clm-3.18.0-x86_64.iso\](https://github.com/abiosoft/alpine-lima/releases/download/colima-v0.5.5/alpine-lima-clm-3.18.0-x86_64.iso/)""
time="2023-09-01T03:49:46Z" level=info msg="[hostagent] Starting QEMU (hint: to watch the boot progress, see \"/Users/runner/.lima/colima/serial*.log\")"
time="2023-09-01T03:49:46Z" level=info msg="SSH Local Port: 49204"
time="2023-09-01T03:49:46Z" level=info msg="[hostagent] Waiting for the essential requirement 1 of 5: \"ssh\""
time="2023-09-01T03:49:46Z" level=info msg="[hostagent] Driver stopped due to error: \"exit status 255\""
time="2023-09-01T03:49:46Z" level=info msg="[hostagent] Shutting down the host agent"
time="2023-09-01T03:49:46Z" level=warning msg="[hostagent] failed to exit SSH master"
time="2023-09-01T03:49:46Z" level=info msg="[hostagent] Shutting down QEMU with ACPI"
time="2023-09-01T03:49:46Z" level=warning msg="[hostagent] failed to open the QMP socket \"/Users/runner/.lima/colima/qmp.sock\", forcibly killing QEMU"
time="2023-09-01T03:49:46Z" level=fatal msg="error starting vm: error at 'creating and starting': exit status 1"
time="2023-09-01T03:49:46Z" level=info msg="[hostagent] QEMU has already exited"
time="2023-09-01T03:49:46Z" level=fatal msg="exiting, status={Running:false Degraded:false Exiting:true Errors:[] SSHLocalPort:0} (hint: see \"/Users/runner/.lima/colima/ha.stderr.log\")"
```

**Comment by squakez** [comment]
We may try the workaround proposed in https://github.com/abiosoft/colima/issues/172

**Comment by squakez** [comment]
The issue on kind is https://github.com/kubernetes-sigs/kind/issues/3277 - we can try to temporarily downgrade using Kind 1.19.

**Comment by squakez** [comment]
This is fixed with a downgrade to Kind 0.19. We'll keep it open to verify it works as soon as the problem is solved in Kind with future versions.
